src/app/interface-form/interface-form.component.html

```diff
@@ -1,5 +1,7 @@
 <div class="form-body">
-  <form [formGroup]="betForm">
+  <form [formGroup]="betForm" (ngSubmit)="onSubmit($event)">
     <app-money-input formControlName="amount"></app-money-input>
+    <app-coefficient [coefficient]="2"></app-coefficient>
+    <app-bet-button [isDisabled]="!betForm.valid"></app-bet-button>
```

Comment:
You also may use `invalid` property instead of inversed `valid`

Reply:
done